agent/setup.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
-# This is an old version, but we know it works, so will not attempt an upgrade right now.
-version=2.302.1
+# This will always get the latest version of the runner binary.
+
+version=`curl -s https://api.github.com/repos/actions/runner/releases/latest | jq -r .tag_name | cut -c 2-`
```

Comment:
Are we sure that we always want the latest? I suppose it is easy to change back to a hardcoded version if we have to.

Reply:
This will probably be better, as it always gets the latest version

Reply:
Let's not depend on jq here.